linkerd/core/src/main/scala/io/buoyant/linkerd/ProtocolInitializer.scala

```diff
@@ -86,9 +87,16 @@ abstract class ProtocolInitializer extends ConfigInitializer { initializer =>
         override def close(d: Time) = Future.Unit
       }
 
-      val servable = servers.map { server =>
-        val stackServer = defaultServer.withParams(defaultServer.params ++ server.params)
-        ServerInitializer(protocol, server.addr, stackServer, adapted, server.announce)
+      val servable = servers.map { s =>
+        val stk = s.params[ClearContext.Param] match {
+          case ClearContext.Param(true) => clearServerContext(defaultServer.stack)
```

Comment:
We only include the module if the param is enabled, and then check the param again in the module itself.  Isn't this redundant?